src/containers/Storage/VDiskPopup/VDiskPopup.tsx

```diff
@@ -51,15 +52,17 @@ const prepareVDiskData = (data: TVDiskStateInfo, poolName?: string) => {
         AllocatedSize,
         ReadThroughput,
         WriteThroughput,
+        StoragePoolName,
     } = data;
 
     const vdiskData: InfoViewerItem[] = [
         {label: 'VDisk', value: stringifyVdiskId(VDiskId)},
         {label: 'State', value: VDiskState ?? 'not available'},
     ];
 
-    if (poolName) {
-        vdiskData.push({label: 'StoragePool', value: poolName});
+    // Depending on endpoint VDisks may have or lack StoragePoolName
+    if (poolName || StoragePoolName) {
```

Comment:
We pass it in StorageGroups. VDisks don't have StoragePoolName property, if they are inside StoragePools data, so the name should be passed separately

Reply:
It seems it will be more straight way to define this value in one place - i suggest to do it here. And in `StorageGroups` we may just enrich vdisk's data.